presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -904,13 +906,23 @@ private String getQueryPlan(Explain node, ExplainType.Type planType, ExplainForm
     {
         switch (planFormat) {
             case GRAPHVIZ:
-                return queryExplainer.get().getGraphvizPlan(session, node.getStatement(), planType);
+                return queryExplainer.get().getGraphvizPlan(session, unwrapStatementToExplain(node), planType);
             case TEXT:
-                return queryExplainer.get().getPlan(session, node.getStatement(), planType);
+                return queryExplainer.get().getPlan(session, unwrapStatementToExplain(node), planType);
         }
         throw new IllegalArgumentException("Invalid Explain Format: " + planFormat.toString());
     }
 
+    private Statement unwrapStatementToExplain(Explain node)
+    {
+        Statement statement = node.getStatement();
+        if (!(statement instanceof Execute)) {
```

Comment:
If we have a method that can take a `Statement`, no need for this check here, which is repeated in `StatementCreator`